src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -224,10 +231,19 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 	public toggleEditMode(editMode?: boolean): void {
 		this.isEditMode = editMode !== undefined ? editMode : !this.isEditMode;
 		this.cellModel.isEditMode = this.isEditMode;
+		if (this.isEditMode === false) {
+			this.togglePreview(true);
+		}
 		this.updatePreview();
 		this._changeRef.detectChanges();
 	}
 
+	public togglePreview(showPreview?: boolean): void {
```

Comment:
Why is the parameter optional? Seems like we should always be passing in an actual boolean

Reply:
I as ... following the pattern used for toggleEditMode.
I don't think it needs to be nullable here. What IS nullable is the modelInterfaces.ts > ICellModel > property showPreview